pkg/imgpkg/imageset/image_set.go

```diff
@@ -97,6 +97,19 @@ func (o *ImageSet) Import(imgOrIndexes []imagedesc.ImageOrIndex,
 			}
 			imageOrIndexesToWriteLock.Lock()
 			defer imageOrIndexesToWriteLock.Unlock()
+
+			itemDigest, err := item.Digest()
+			if err != nil {
+				errCh <- err
+				return
+			}
+			importDigestRef, err := regname.NewDigest(fmt.Sprintf("%s@%s", importRepo.Name(), itemDigest))
```

Comment:
the logic for determining the destination ref is elsewhere (one of the other private functions). having it be in two distinct places doesnt smell right to me.

Reply:
@DennisDenuto and I took a look and we couldn't identify a private function that determined the destination digest reference of an image. We found one for returning a tag, but we want to print out the digest. The diff shows that we pulled the logging statement from tagAndVerifyItem(), so we copied the prerequisites from that function for the logging.